src/org/omegat/filters3/xml/xliff/XLIFFDialect.java

```diff
@@ -160,6 +166,47 @@ public Boolean validateContentBasedTag(String tag, Attributes atts) {
         return "mrk".equals(tag) && atts != null && "protected".equals(atts.getValueByName("mtype"));
     }
 
+    /**
+     * When translation is different from src, transit <target state="needs-translated"> attr
+     * to <target state="translated"> value.
+     * https://sourceforge.net/p/omegat/feature-requests/1506/
```

Comment:
I think for Javadoc purposes we need

- To escape `<` (and `>`?) as HTML entities: `&lt;` (and `&gt;`?)
- Put the link in a `@see <a href="...">...</a>`